src/main/java/com/google/sps/data/PrioritizeCoursesCriteria.java

```diff
@@ -24,7 +24,7 @@ public float preferenceScore(Schedule schedule) {
     // Instantiate the HashMap, courseScores
     for (int i = 0; i < courseList.size(); i++) {
       Course currentCourse = courseList.get(i);
-      for(Section section: currentCourse.getSections()){
+      for(Section section: currentCourse.getlectureSections()){
```

Comment:
`getlectureSections` --> `getLectureSections`

Reply:
Sounds good I made the change